packages/web3-eth/src/index.js

```diff
@@ -483,6 +483,11 @@ var Eth = function Eth() {
             params: 0,
             outputFormatter: utils.hexToNumber
         }),
+        new Method({
+            name: 'requestAccounts',
+            call: 'eth_requestAccounts',
+            params: 0
```

Comment:
`eth_accounts` [output-formats to checksum address][1].  Maybe the same here? 


[1]: https://github.com/ethereum/web3.js/blob/5a08984ca2c80e30bc59e429692b0b5960274493/packages/web3-eth/src/index.js#L338

Reply:
True that 👌